spec/system/support_interface/exporting_applications_for_ucas_spec.rb

```diff
@@ -36,12 +36,12 @@ def and_i_visit_the_service_performance_page
     visit support_interface_performance_path
   end
 
-  def when_the_feature_is_not_enabled
-    FeatureFlag.deactivate('download_dataset1_from_support_page')
+  def when_i_am_on_production
+    ENV['HOSTING_ENVIRONMENT_NAME'] = 'production'
```

Comment:
I think this and the other `ENV` assignment are unsafe because they might hang around once the test is done? `ClimateControl` could do the cleanup for us.

Reply:
Can I use `ClimateControl` without a block? Not sure how I'd use it otherwise in these tests.

Reply:
I don't think so — we'd have to indent in the body of the spec.

How about something like this?

```ruby
around do |e|
  existing_env = ENV
  e.run
  ENV = existing_env
end
```

Reply:
I've checked, and tests that rely on a specific value already use climate control or set it explicitly, so I think this actually won't cause an issue?